### PR TITLE
Turned down global diagnostic rate to 1Hz

### PR DESCRIPTION
**Pull request by ipa-mig**

### CODE REVIEW

**Comment by ipa-mig** [comment]
@mig-em looks good. Will test it on the robot.

File: cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp [hunk 1]
```diff
@@ -714,38 +717,48 @@ class NodeClass
...
+                    if (m_bisInitialized)
+                    {
+                      diagnostics_gl.status[0].level = 0;
+                      diagnostics_gl.status[0].name = ros::this_node::getName(); //"schunk_powercube_chain";
+                      diagnostics_gl.status[0].message = "base_drive_chain initialized and running";
+                    }
+                    else
+                    {
+                      diagnostics_gl.status[0].level = 1;
+                      diagnostics_gl.status[0].name = ros::this_node::getName(); //"schunk_powercube_chain";
```
Comment (ipa-mig) [inline cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp]: maybe remove those  `schunk` comments, though...

**Comment by mig-em** [comment]
@ipa-mig Should I test this?

**Comment by ipa-mig** [comment]
yes please. Maybe talk to flg about whether the robot can be used right now...

**Comment by ipa-mig** [comment]
merged in ipa320/cob_driver#329

**Pull request closed without merging**